### ISSUE TITLE
Make psammead-bulleted-list more accessible by updating element roles

### ISSUE THREAD

**Issue by pharingee** [BBC-archive/psammead#2766]
**Is your feature request related to a problem? Please describe.**
There's a bug on VoiceOver that does not announce the list when `list-style-type` is set to none.

**Describe the solution you'd like**
Add `role='list'` and `role='list item'` attributes to `ul` and `li` elements respectively.

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
